File: sdks/go/blueprint.go
package blueprint

// Blueprint represents the Blueprint SDK
type Blueprint struct {
    version string
}

// New creates a new Blueprint instance
func New() *Blueprint {
    return &Blueprint{
        version: "1.0.0",
    }
}

// Hello returns a greeting message
func (b *Blueprint) Hello() string {
    return "Hello from Blueprint SDK!"
}

// Version returns the SDK version
func (b *Blueprint) Version() string {
    return b.version
}

File: sdks/csharp/Blueprint.cs
using System;

namespace Blueprint.Sdk
{
    /// <summary>
    /// Blueprint SDK for C#
    /// </summary>
    public class Blueprint
    {
        private string version = "1.0.0";
        
        /// <summary>
        /// Returns a greeting message
        /// </summary>
        public string Hello()
        {
            return "Hello from Blueprint SDK!";
        }
        
        /// <summary>
        /// Returns the SDK version
        /// </summary>
        public string GetVersion()
        {
            return version;
        }
    }
}
